Firmware/ultralcd.cpp

```diff
@@ -2992,6 +2992,7 @@ bool lcd_calibrate_z_end_stop_manual(bool only_z)
             if (lcd_clicked()) {
                 // Abort a move if in progress.
                 planner_abort_hard();
+                planner_aborted = false;
```

Comment:
Maybe it is better to add this change in a separate commit so it is easier to cherry pick into 3.12. The `LCD_LEFT_BUTTON_CHOICE` enum isn't currently in 3.12.

Reply:
Done